random_generator/utils/atl_functions.py

```diff
@@ -0,0 +1,112 @@
+from enum import IntEnum
+
+import numpy as np
+from utils.configuration import Config
+from utils.utility_functions import write_json
+
+
+class Enforce(IntEnum):
+    enforce = 0
+    despite = 1
+
+    def __str__(self):
+        return self.name
+
+
+class WhateverThisIsCalled(IntEnum):
```

Comment:
TemporalOperator? xD

Reply:
Ha yep, this was one of those "I need to remember to change this before creating a PR".